LICENSE

```diff
@@ -1,6 +1,6 @@
-The MIT License (MIT)
+MIT License
 
-Copyright (c) 2016 Kye Hohenberger
+Copyright (c) Emotion team and other contributors
```

Comment:
React doesn't include the year - so I thought it's good to just drop it. If they can do it - we can also 😅 

I was not entirely sure who to put as copyright holders, chosen "Emotion team and other contributors". If you want to change this, let me know.